content/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs.md

```diff
@@ -288,6 +288,10 @@ jobs:
         run: echo "Running PR only CI"
 ```
 
+### Note on `github.repositoryUrl` property
+
+The `git://` protocol is disabled for security reasons. You will not be able to use the value of `github.repositoryUrl` to perform any git operations. Instead, you can use `${{ github.serverUrl }}${{ github.repository }}` to give you the HTTP URL for the current repository.
+
```

Comment:
Thanks for the clear note. In this case, I think that users will find the information more easily if we put the information directly in the table, as suggested above.